src/plugins/github/parseReferences.js

```diff
@@ -46,20 +41,5 @@ export function findGithubUrlReferences(body: string): GithubUrlMatch[] {
       /(?:\W|$)/.source,
     "gm"
```

Comment:
Unrelated to this change, but I think that you do not actually need the `m` (multiline) flag, because `/\n|\r/` is a subset of `/\W/`. Tests pass without the flag. If you think that the flag should be there, then I would like to see a test to this effect (in a different PR).

Reply:
Ack